ginkgo.Jenkinsfile

```diff
@@ -85,7 +85,7 @@ pipeline {
                     // rm -rf ${GOPATH}/src/github.com/cilium/cilium
                     sh 'cd test/; ./post_build_agent.sh || true'
                     sh 'cd test/; ./archive_test_results.sh || true'
-                    archiveArtifacts artifacts: "test_results_${JOB_BASE_NAME}_${BUILD_NUMBER}.zip", allowEmptyArchive: true
+                    archiveArtifacts artifacts: "test_results_${JOB_BASE_NAME}_${BUILD_NUMBER}.tar", allowEmptyArchive: true
```

Comment:
Why is this a .tar? shouldn't this be kept a zip file so we can easily upload it to a GH-Issue?

Reply:
That feature was introduced after 1.0 was released.

Reply:
Can't we backport that feature to 1.0?

Reply:
I don't think that add a real value, the data is in there and the backport will be difficult. IMHO.